seed/static/seed/partials/inventory_detail_analyses.html

```diff
@@ -40,7 +40,9 @@ <h2>
                 <b>{$:: type $}</b> <small>({$:: analyses.length $})</small>
             </uib-tab-heading>
             <div class="tab-content-cards">
-                <div ng-repeat="analysis in analyses" ng-include="::urls.static_url + 'seed/partials/analysis_card.html'" class="tab-content-card">
+                <!-- NOTE: This is bad, using a controller created for someone else but it will work
+                     we should refactor this into a AnalysisCard component. -->
+                <div ng-repeat="analysis in analyses" ng-controller="analysis_details_controller" ng-include="::urls.static_url + 'seed/partials/analysis_card.html'" class="tab-content-card">
```

Comment:
This is bad! I'm using a controller that's really written for the analysis_details.html partial. But it works well enough here without writing another controller that does basically the same thing. We should probably refactor these partials (analysis_details.html and analysis_card.html) into components so that they can do their own thing more cleanly